README.md

```diff
@@ -0,0 +1,73 @@
+# Blackjack Card Game ♦️♣️♥️♠️
+![picture of starting screen of my game](./images/blackjackRM.png)
+# Play the Game
+Hardcoded for 1600x900 at the moment:
+- [https://euril-blackjack.surge.sh/]
+<!-- start of description -->
+# Blackjack 🂮🃁
```

Comment:
Suggested edits for this section:

My game is a somewhat faithful representation of Blackjack. The goal of the game is to beat the dealer by getting as close to a total of 21 without going over 21. Each card of a standard 52 card deck has a value. Number cards have their own respective value. Face cards are worth 10, and the Ace is worth 1 or 11. 

The player goes first, drawing cards to get as close to 21, then the dealer does the same thing. The two then compare the total value of their drawn cards, and whoever is closer to 21 wins. However, when drawing cards, if the total goes over 21, you immediately lose "Bust". The rules are simple, the play is thrilling, and there is the opportunity for high strategy.

At the start of each round, the player makes a bet of 100p, 500p, or 1000p. From there, the game is played. The player gets 2 cards, and the dealer gets 2 cards with the second card back-side faceup obscuring its value. The player then gets to either Hit, Double, or Stand.

- Hit: Player draws 1 card from the deck and adds that value to their hand. The player can continue hitting as long as the hand value doesn't go beyond 21.
- Double: Only selectable when the first two drawn cards total equal to 9, 10, or 11. When selected, you double your bet and only draw 1 card to add to the hand.
- Stand: The player ends their turn.

After the player turn, the dealer then gets their turn. The difference is that the dealer must keep hitting if they are under or equal to 16 and must stand above or equal to 17. Once both player and dealer get a turn and neither has gone beyond 21 the totals are compared. Whoever is closer to 21 is the winner. Bets will then be paid out depending on who won, the board will be cleared, and any used cards will be discarded.

My Blackjack game uses 6 decks for a total of 312 cards. Once below 50%, the deck will be reshuffled.